packages/telemetry/lib/import-or-local.js

```diff
@@ -0,0 +1,13 @@
+const { pathToFileURL } = require('node:url')
+const resolve = require('resolve')
```

Comment:
why are you importing this? There is no need. You can use `createRequire` from node.js core.